python/ray/tests/test_output.py

```diff
@@ -108,7 +108,7 @@ def test_autoscaler_infeasible():
 import ray
 import time
 
-ray.init(num_cpus=1)
+ray.init(num_cpus=1, num_gpus=0)
```

Comment:
I am not sure how tests on master are passing, it seems somehow the autoscaler will look for available GPUs if not expressly forbidden to do so?

Reply:
Hmm what's the issue? Unless the test instance has GPU (which we shouldn't), num_gpus should be automatically 0

Reply:
Should the test fail if run on a user machine with a GPU?